protocol/message.go

```diff
@@ -106,6 +181,14 @@ func NewKeyLookupProof(ap *m.AuthenticationPath, str *m.SignedTreeRoot,
 	}, e
 }
 
+// NewKeyLookupInEpochProof creates the response message a CONIKS server
+// sends to a client upon a KeyLookupInEpochRequest.
+// The contained DirectoryProofs is either a single proof of inclusion
+// (e = Success), or a proof of absence if the requested name does not
+// exist in the directory (e = ErrorNameNotFound).
+// In either case, the server returns a list of STRs for the epoch range [ep,
+// directory.LatestEpoch().Epoch], where ep is the past epoch for which
+// the client has requested the user's key.
```

Comment:
Thinking aloud here: I wonder if it wouldn't make more sense to move the actual behaviour of the server to methods where it actually happens (for instance `KeyLookupInEpoch` creates the list of STRs which are, then, passed to this constructor)